xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-ui/src/main/resources/XWiki/DocumentsMacro.xml

```diff
@@ -183,11 +183,12 @@ Generates:
   'rowCount' : $xcontext.macro.params.count,
   'description' : 'This table lists documents found on this wiki based on passed criteria. The columns can be sorted and some can be filtered.'
 })
-#if ("$!xcontext.macro.params.space" != "") 
-  #set ($extraParams = "&amp;space=$escapetool.url($xcontext.macro.params.space)")
-#end
-#if ("$!xcontext.macro.params.parent" != "") 
-  #set ($extraParams = "${extraParams}&amp;parent=$escapetool.url($xcontext.macro.params.parent)")
```

Comment:
Actually, the fix is to just add a missing `!` to `${extraParams}`, resulting in `$!{extraParams}`. This way, if *only* the parent is specified and `${extraParams}` is null, it will append an empty string (` `) instead of `${extraParams}`.